### DISCUSSION TITLE
Cartoon Network Pakistan

### DISCUSSION THREAD

**Discussion by spongebob849** [iptv-org/community#763] [Q&A]
### Channel Name

Cartoon Network Pakistan

### Channel ID

CartoonNetwork.pk

### Country

Pakistan

### Language

English, Urdu

### Website

https://www.cartoonnetwork.com/

### Notes

https://en.wikipedia.org/wiki/Cartoon_Network_(Pakistani_TV_channel)

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19764
